packages/graph/hash_graph/tests/test_data/link_type/owns_v1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "kind": "linkType",
+  "$id": "https://blockprotocol.org/@alice/types/link-type/owns/v/1",
+  "title": "Owns",
+  "description": "Have (something) as one's own; possess",
```

Comment:
Maybe it's worth testing a link type without a description. As we have `"Owns"` v2, this is probably a good place.

Reply:
Do we have broken types anywhere else, or is this one you want to break for the fun of it? 😁 

Description is not optional

Reply:
> Description is not optional

I haven't noticed that, thanks for the clarification!